### ISSUE TITLE
Widget only able to set 50% of the screens possible brightness

### ISSUE THREAD

**Issue by veehexx** [davidhi7/ddcci-plasmoid#80]
i've 2 different monitors connected and able to control both.
One of them (Eizo FS2333) is not adjusting brightness correctly where the UI slider bar is at 100%, the monitor is only at 50% of it's brightness.
adjusting brightness on the screen then makes your plasmoid widget report >100%. setting the screen to 100% and forcing a rescan made it crash the entire kde launcher.

hopefully i've ran the correct command to get a possible fix figured out. current brightness according to the screen is 87%, with your widget reporting 175%:
[eizo-fs2333 ddcutil probe.txt](https://github.com/davidhi7/ddcci-plasmoid/files/15047921/eizo-fs2333.ddcutil.probe.txt)

**Comment by davidhi7** [comment]
Your monitor appears to accept brightness values between 0 (?) and 200 according to the ddcutil output. Could you check this behaviour by running `ddcutil setvcp --bus 18 10 0` and `ddcutil setvcp --bus 18 10 200` (update the bus id if neccessary) and report whether these commands correctly set the monitor brightness to the lowest and highest possible value, respectively?

The widget assumes brightness values reported from and accepted by the monitor are always between 0 and 100. Fixing this should be trivial.

**Comment by Tau512** [comment]
yes - works as expected; `ddcutil setvcp --bus 18 10 0` sets to 0%, `ddcutil setvcp --bus 18 10 200` sets to 100% brightness.
